themes/startwords/layouts/article/single.html

```diff
@@ -1,3 +1,9 @@
+{{ define "css" }}
+    {{ range .Resources.Match "*.css" }}
+    <link rel="stylesheet" href="{{ .RelPermalink }}">
+    {{ end }}
+{{ end }}
+
```

Comment:
This is a great solution! We should make sure to document how to add page-specific css somewhere (readme?).

I wonder if this could be added to the base template so that it would apply to any page and not have to be in both article and issue layouts?

Reply:
I wondered about that too, but wasn't sure what the actual implications would be...for instance, if I dropped a `style.css` file into the entire `content/` root...what would happen? It seemed like there might be a potential for confusing behavior to occur, and I also didn't make any provisioning for ensuring that multiple files cascaded in the correct order.

Reply:
I thought that the resources were specific to page based on our experience with the images earlier, but I think you've worked with this more and I trust your inclination.